packages/replay-next/components/sources/log-point-panel/LogPointPanel.tsx

```diff
@@ -166,6 +168,16 @@ function PointPanelWithHitPoints({
     };
   }
 
+  let source = getSource(client, point.location.sourceId);
+  if (source?.kind === "prettyPrinted") {
+    assert(
+      source.generatedSourceIds,
```

Comment:
This just asserts that there's an array, right? Not that it's non-empty?

Reply:
Yeah, that was more about appeasing typescript.